onnxruntime/python/onnxruntime_inference_collection.py

```diff
@@ -226,6 +226,35 @@ def run(self, output_names, input_feed, run_options=None):
                 return self._sess.run(output_names, input_feed, run_options)
             raise
 
+    def run_async(self, output_names, input_feed, callback, user_data, run_options=None):
+        """
+        Compute the predictions asynchronously in a separate thread
+
+        :param output_names: name of the outputs
+        :param input_feed: dictionary ``{ input_name: input_value }``
+        :param callback: python function that accept array of results, and an status string on error
```

Comment:
Also, mention that this callback will be called by one of the intra op threads spawned by the internal C++ runtime.

Reply:
Done.